js/packages/for-native/src/mixpanel_backend.ts

```diff
@@ -0,0 +1,56 @@
+import Mixpanel from "react-native-mixpanel"
+import { UIBackend } from "@utilitywarehouse/customer-tracking-core"
+import { Actor } from "@utilitywarehouse/customer-tracking-types"
+
+export class MixpanelBackend implements UIBackend {
+    constructor(token: string, options: {[k: string]: string | number | boolean} = {}) {
+        const {
+            optOutTrackingByDefault,
+            trackCrashes,
+            automaticPushTracking,
+            launchOptions,
+        } = options
+        Mixpanel.sharedInstanceWithToken(
```

Comment:
Is there no way to not have shared instance?

Reply:
doesn't look like it even in the native docs...
https://developer.mixpanel.com/docs/ios